### ISSUE TITLE
[Bug] Refresh on drag - Client

### ISSUE THREAD

**Issue by Liyf4TakingLs** [TekExplorer/featherdactyl.com#40]
**Description**
There is a refresh button, but it would be nice to be able to refresh by dragging down with your finger.

**To Reproduce**
```
1. Try to refresh when you do not have many servers
```
**Expected behavior**
Upon dragging down it should refresh the server status. Currently it only works if you have few servers and the scrollbar appears 

**App (please complete the following information):**
```
- Version: v0.0.22
- Build: 224
```
**Smartphone (please complete the following information):**
```
 - Device: Samsung Galaxy S20 
 - Model: SM-G980F/DS
 - OS: Android 12
```

**Comment by drjayy** [comment]
This should work inside the app, unless Tek removed it or messed something a little bit up

However I'll try to confirm in a second.

**Comment by TekExplorer** [comment]
> **EDIT:** Apparently Tek removed this function
Not deliberately. Not sure why it doesn't activate, but its certainly one of the things i want to fix

**Comment by TekExplorer** [comment]
this should be fixed as of `0.0.25`
please reopen if it is not.

turns out, it works, but only if you had enough stuff to scroll. found a way to always enable scroll.
